src/kili/graphql/operations/issue/queries.py

```diff
@@ -0,0 +1,52 @@
+"""
+GraphQL Queries of Issues
+"""
+
+from typing import Optional
+
+from kili.graphql import BaseQueryWhere, GraphQLQuery
+from kili.types import Issue
+
+
+class IssueWhere(BaseQueryWhere):
+    """
+    Tuple to be passed to the IssueQuery to restrict query
+    """
+
+    # pylint: disable=too-many-arguments,too-many-locals,too-many-instance-attributes
```

Comment:
we can remove that flag

@PierreLeveau we should consider removing those pylint flags (too-many-...). We ignore them anyway, so better disabling them from linting settings maybe

Reply:
i would say it depends... For internal code, especially the core, I think it is interesting to keep them as it underlines good practices. For external code, it is almost a design choice to pass flat argument lists so we should silence them.

Reply:
We already discussed this. I think that Pierre was in favor of keeping them so it applies to internal functions and we keep it for user-facing functions